style.css

```diff
@@ -0,0 +1,102 @@
+.sort {
+    width: 8px;
+}
+
+.heart {
+    width: 25px;
+}
+
+.user {
+    width: 22px;
+    opacity: 0.7;
+}
+
+/* .heart:hover {
+    height: 30px;
+    width: 25px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+  } */
+
+header {
+    max-width: 1200px;
+    height: 80px;
+    margin-left: 31.667px;
+    margin-right: 31.667px;
+}  
+
+nav{
+    padding-top: 20px;
+    padding-bottom: 20px;
+    color: #053361;
+}
+
+header a{
+    color: #000000;
+    text-decoration: none;
+    font-size: 16px;
+}
+
+header ul{
+    margin: 0;
+    padding: 0;
+}
+
+.right{
+    float: right;
+
+}
+
+.right img {
+    margin: 0 0 0 5px;
+}
+
+.right a {
+    margin: 0 0 0 9px;
+}
+
+.left{
+    float: left;
+}
+
+.left a {
+    margin: 0 0 0 9px;
+}
+
+.header {
+    font-size:35px;
+    padding-top: 5px;
+    font-weight: bold;
+    color: #053361;
+}
+
+.search-bar{
+    height: 25px;
+    width: 1200px;
+    padding: 10px;
+    background: #f2f2f2;
+    border: black;
+    border-radius: 4px;
+    margin-bottom: 18px;
+}
+
+.search-btn{
+    color: black;
+    padding-top: 6px;
+    float: right;
+    width: 20px;
+    border-radius: 50%;
+}
+
+input{
+    padding-top: 5px;
+    padding-bottom: 5px;
+    color: #2a343d;
+    font-size: 16px;
+    width: 500px;
+    background: none;
+    border: none;
+    font-family: roboto;
+    font-size: 18px;
+}
```

Comment:
@Afrocentriccynic From the screenshots, you can see that the padding, font-sizes do not match.

<img width="1440" alt="Screenshot 2020-05-10 at 6 40 18 PM" src="https://user-images.githubusercontent.com/16891274/81506400-eda2df00-92ed-11ea-8ced-9dbb7075a963.png">

<img width="1440" alt="Screenshot 2020-05-10 at 6 41 34 PM" src="https://user-images.githubusercontent.com/16891274/81506402-f0053900-92ed-11ea-840b-195c90576861.png">